docs/learn/learn-staking.md

```diff
@@ -1,324 +1,390 @@
+
+Two different accounts can be used to securely manage your funds while staking.
 
 - **Stash:** This account holds funds bonded for staking, but delegates some functions to a
-  Controller. As a result, you may actively participate with a Stash key kept in a cold wallet,
-  meaning it stays offline all the time. You can also designate a Proxy account to vote in
-  [governance](learn-governance.md) proposals.
-- **Controller** This account acts on behalf of the Stash account, signalling decisions about
-  nominating and validating. It sets preferences like payout account and commission. If you are a
-  validator, it also sets your [session keys](learn-keys.md#session-keys). It only needs enough
-  funds to pay transaction fees.
+  controller account. As a result, you may actively participate in staking with a stash private key
+  kept in a cold wallet like Ledger, meaning it stays offline all the time. Stash account keys are
+  used to sign staking actions such as bonding additional funds.
+
+- **Controller:** This account acts on behalf of the stash account, signaling decisions about
+  nominating and validating. It can set preferences like commission (for validators) and the staking
+  rewards payout account. The earned rewards can be bonded (locked) immediately for bonding on your
+  stash account, which would effectively compound the rewards you receive over time. You could also
+  choose to have them deposited to your controller account or a different account as a free
+  (transferable) balance. If you are a validator, it can also be used to set your
+  [session keys](learn-keys.md#session-keys). Controller accounts only need sufficient funds to pay
+  for the transaction fees.
+
+:::warning
+
+Never leave a high balance on a controller account which are usually "hot" as their private key is
+stored on the device (PC, phone) and it is always exposed to the internet for potential hacks and
+scams. It is good practice to deposit rewards on the stash account or to send them to another
+account on a cold wallet.
+
+:::
+
+![staking](../assets/staking/stash-controller.png)
+
+This hierarchy of separate keys for stash and controller accounts was designed to add a layer of
+protection to nominators and validator operators. The more often one exposes and uses a private key,
+the higher its vulnerability for hacks or scams. So, if one uses a key for multiple roles on a
+blockchain network, it is likely that the account can get compromised. Note that the damage linked
+to stolen private keys is different depending on the type of account derivation. In the case of soft
+derivation, all derived accounts are compromised. More information about account derivation can be
+found [here](../learn/learn-accounts.md/#derivation-paths).
+
+:::info
+
+For Ledger users staking directly on Ledger Live, currently, there is no option to use separate
+stash and controller accounts. That is if you stake on Ledger Live your stash account will be your
+controller too.
```

Comment:
Yes this is an issue. We might want to add that Ledger is now supported in Talisman extension. Users can import their Ledger accounts in there and use them as a stash and controller.

Ledger still doesn't support key extrinsics like identity though (2022, shocking!) which is why I personally hold off from using Ledger.

Reply:
Added